tech/languages/nodejs/modules.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Modules
```

Comment:
Modules seems a little too general even though I understand they are called just modules. Can we make the title a little more specific? e.g. Node.js Modules